### PR TITLE
Unifi fixes

### PR DESCRIPTION
**Pull request by colemamd**
Minor fixes to unifi. Tested working with Grafana as well

### CODE REVIEW

File: blueprints/unifi/install.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ UP_USER="${!UP_USER:-$1}"
 UP_PASS="jail_${1}_up_password"
 INCLUDES_PATH="${SCRIPT_DIR}/blueprints/unifi/includes"
 
-if [ -z "${!DB_PASSWORD}" ]; then
+if [ -z "${DB_PASS}" ]; then
```
Comment (Ornias1993) [inline blueprints/unifi/install.sh:37]: Shoot my mistake.
Still needs to be:
```suggestion
if [ -z "${!DB_PASS}" ]; then
```

File: blueprints/unifi/install.sh [hunk 2]
```diff
@@ -70,7 +70,7 @@ else
   # Check if influxdb container exists, create unifi database if it does, error if it is not.
   echo "Checking if the database jail and database exist..."
   if [[ -d /mnt/"${global_dataset_iocage}"/jails/"${!DB_JAIL}" ]]; then
-    DB_EXISTING=$(iocage exec "${!DB_JAIL}" curl -G http://localhost:8086/query --data-urlencode 'q=SHOW DATABASES' | jq '.results [] | .series [] | .values []' | grep "$DB_NAME" | sed 's/"//g' | sed 's/^ *//g')
+    DB_EXISTING=$(iocage exec "${!DB_JAIL}" curl -G http://"${DB_IP}":8086/query --data-urlencode 'q=SHOW DATABASES' | jq '.results [] | .series [] | .values []' | grep "$DB_NAME" | sed 's/"//g' | sed 's/^ *//g')
```
Comment (Ornias1993) [inline blueprints/unifi/install.sh:73]: SHOOT, thanks!

File: blueprints/unifi/install.sh (hunk 1 above)
Comment (colemamd) [reply blueprints/unifi/install.sh]: yea, my bad. fixed

**Comment by Ornias1993** [comment]
Quick fix, quick merge. Thanks!

**Pull request merged**